panels/search/cc-search-panel.c

```diff
@@ -677,18 +677,28 @@ search_providers_discover_thread (GTask *task,
                                   GCancellable *cancellable)
 {
   g_autoptr(DiscoverProvidersResult) result = NULL;
+  g_autoptr(GDBusConnection) system_bus = NULL;
+  MctManager *manager;
+  g_autoptr(MctAppFilter) filter = NULL;
   GList *providers = NULL;
   const gchar * const *system_data_dirs;
   int idx;
-  g_autoptr(EpcAppFilter) filter = NULL;
   GError *local_error = NULL;
 
+  system_bus = g_bus_get_sync (G_BUS_TYPE_SYSTEM, cancellable, &local_error);
+  if (system_bus == NULL)
+    {
+      g_task_return_error (task, g_steal_pointer (&local_error));
+      return;
+    }
+
+  manager = mct_manager_new (system_bus);
   /* Load the user’s parental controls settings too, so we can filter the list. */
-  filter = epc_get_app_filter (NULL,
-                               getuid (),
-                               FALSE,
-                               cancellable,
-                               &local_error);
+  filter = mct_manager_get_app_filter (manager,
+                                       getuid (),
+                                       MCT_GET_APP_FILTER_FLAGS_INTERACTIVE,
```

Comment:
The use of the `INTERACTIVE` flag here is a behaviour change from the old code. The old code passed `FALSE` to the `allow_interactive_authorization` argument of `epc_get_app_filter()`.